internal/chart/helm_client.go

```diff
@@ -91,6 +94,10 @@ func (c HelmClient) UpdateChart(tv TemplateValuer, config ChartConfig, opts ...I
 	}
 	updateClient := action.NewUpgrade(c.cfg)
 	updateClient.Namespace = c.namespace
+	// MaxHistory=0 means there is an unlimited number of k8s secrets per application,
```

Comment:
We might need to update comments, we are only mentioning `secrets` here but helm docs say:

```
MaxHistory specifies the maximum number of historical releases that will be retained, including the most recent release.
Values of 0 or less are ignored (meaning no limits are imposed).

```